interlok-oauth-salesforce/src/test/java/com/adaptris/core/oauth/salesforce/SalesforceLoginWorkerTest.java

```diff
@@ -168,7 +169,7 @@ public void testCustomResponseHandler() throws Exception {
     handler.throwExceptionIfAny();
   }
 
-  @Test(expected = HttpResponseException.class)
+  @Test()
```

Comment:
You don't need to keep () after @Test

Reply:
gahhh silly mistake and didn't remove it from this one! Removed now.